### PR TITLE
Issue Resolution: Implement Events Section & Quality Changes

### PR DESCRIPTION
**Pull request by Saransh-Jainbu**
## Important Checklist:
- [✔️ ] You have prettyfied your code (`npx prettier --write .`)
- [✔️] Linted the code using ESLint (`npx eslint . --ext .js,.jsx,.ts,.tsx`)
- [✔️ ] This code is best you could write in terms of quality (we don't mind even if its bad :P)
- [✔️ ] You have attached the screenshot of visible chnages in GUI, if any
- [✔️ ] You have no merge conflicts, you have pulled and rebased your PR branch
- [✔️ ] All CI tests have passed

### CODE REVIEW

File: src/app/page.js [hunk 1]
```diff
@@ -1,15 +1,30 @@
 "use client";
 
+import { useEffect, useState } from "react"; // Import useEffect and useState
 import Navbar from "@/components/Navbar";
-import Links from "../components/Links";
+import EventsSection from "@/components/EventsSection"; // Import EventsSection, which includes EventCard and Events
```
Comment (g4ze) [inline src/app/page.js:5]: weird comm

File: src/app/globals.css [hunk 2]
```diff
@@ -25,6 +25,16 @@ body {
     text-wrap: balance;
   }
 }
+/* Add this in your CSS file */
```
[the remark this answers is not in the document]
Comment (g4ze) [reply src/app/globals.css:28]: still unresolved

**Review by g4ze — CHANGES_REQUESTED** [review]

**Comment by g4ze** [comment]
> [✔️ ] You have attached the screenshot of visible chnages in GUI, if any
unattended

**Comment by Saransh-Jainbu** [comment]
please refer the issue comment for image changes

**Comment by g4ze** [comment]
Kindly attach an image here as well pls

**Review by g4ze — APPROVED** [review]

**Pull request merged**